src/profile/components/ProfilePage.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { StatusAlert, Hyperlink } from '@edx/paragon';
 import { connect } from 'react-redux';
-import { injectIntl, intlShape } from '@edx/frontend-i18n'; // eslint-disable-line
+
 import { sendTrackingLogEvent } from '@edx/frontend-analytics';
+import { App, AuthenticationContext } from '@edx/frontend-base';
+import { injectIntl, intlShape } from '@edx/frontend-i18n'; // eslint-disable-line
```

Comment:
I think we can lose the `// eslint-disable-line` now. I think this was from when i18n live inside the repo as an alias.